tests/flow/tests_pytorch.py

```diff
@@ -928,3 +929,36 @@ def test_pytorch_model_rdb_save_load(env):
     env.assertTrue(dtype_memory == dtype_after_rdbload)
     env.assertTrue(shape_memory == shape_after_rdbload)
     env.assertTrue(data_memory == data_after_rdbload)
+
+
+def test_parallelism():
+    env = Env(moduleArgs='INTRA_OP_PARALLELISM 1 INTER_OP_PARALLELISM 1')
+    if not TEST_PT:
+        env.debugPrint("skipping {} since TEST_PT=0".format(sys._getframe().f_code.co_name), force=True)
+        return
+
+    con = env.getConnection()
+
+    test_data_path = os.path.join(os.path.dirname(__file__), 'test_data')
+    model_filename = os.path.join(test_data_path, 'pt-minimal.pt')
+    with open(model_filename, 'rb') as f:
+        model_pb = f.read()
+
+    ret = con.execute_command('AI.TENSORSET', 'a{1}', 'FLOAT', 2, 2, 'VALUES', 2, 3, 2, 3)
+    ret = con.execute_command('AI.TENSORSET', 'b{1}', 'FLOAT', 2, 2, 'VALUES', 2, 3, 2, 3)
+    ret = con.execute_command('AI.MODELSET', 'm{1}', 'TORCH', DEVICE, 'BLOB', model_pb)
+    ensureSlaveSynced(con, env)
+    con.execute_command('AI.MODELRUN', 'm{1}', 'INPUTS', 'a{1}', 'b{1}', 'OUTPUTS', 'c{1}')
+    ensureSlaveSynced(con, env)
+    values = con.execute_command('AI.TENSORGET', 'c{1}', 'VALUES')
+    env.assertEqual(values, [b'4', b'6', b'4', b'6'])
+    load_time_config = {k.split(":")[0]: k.split(":")[1]
+                        for k in con.execute_command("INFO MODULES").decode().split("#")[3].split()[1:]}
+    env.assertEqual(load_time_config["ai_inter_op_parallelism"], "1")
+    env.assertEqual(load_time_config["ai_intra_op_parallelism"], "1")
```

Comment:
@hhsecond can you return it from `get_num_interop_threads`?
returning from the load_time_config params is no guarantee for it to be actually set

Reply:
As discussed over slack, I'll wait for the `AI.INFO` changes you proposed - I think it is a nice addition. Merging this PR for now